src/app.js

```diff
@@ -383,6 +384,25 @@ function run () {
   })
   var offsetToLineColumnConverter = new OffsetToLineColumnConverter(compiler.event)
 
+  // ---------------- ContextualListener -----------------------
+  this._components.contextualListener = new ContextualListener({
+    getCursorPosition: () => {
+      return this._components.editor.getCursorPosition()
+    },
+    getCompilationResult: () => {
+      return compiler.lastCompilationResult
+    },
+    warnExpression: (position) => {
+      position = offsetToLineColumnConverter.offsetToLineColumn(position, position.file, compiler.lastCompilationResult)
+      return editor.addMarker(position, config.get('currentFile'), 'highlightcall')
```

Comment:
Why highlightcall? Should be something like "highlightCurrentDeclaration" or "...CurrentRefereces"

Reply:
yes right